docs/api.md

```diff
@@ -56,4 +57,6 @@ Public utility functions used elsewhere in the package.
     utils.experiments.fix_broken_second_line
     utils.experiments.compare_result_file_resample
     utils.experiments.assign_gpu
-```
+    utils.experiments.assign_gpu
+</old_file>
+\n
```

Comment:
The lines 61 and 62 are useless and should be removed. Replace them with an empty newline.